pkg/volume/glusterfs/glusterfs.go

```diff
@@ -968,6 +969,11 @@ func getClusterNodes(cli *gcli.Client, cluster string) (dynamicHostIps []string,
 			return nil, fmt.Errorf("failed to get host ipaddress: %v", err)
 		}
 		ipaddr := dstrings.Join(nodeInfo.NodeAddRequest.Hostnames.Storage, "")
+		// IP validates if a string is a valid IP address.
+		ip := net.ParseIP(ipaddr)
+		if ip == nil {
+			return nil, fmt.Errorf("Glusterfs ipAddress %s must be a valid IP address, (e.g. 10.9.8.7)", ipaddr)
```

Comment:
May be we can reword the error as "glusterfs server node ip address %s must be a valid IP address, (e.g. 10.9.8.7)"

Reply:
thanks, done!